src/main/java/com/iexec/common/replicate/ReplicateStatus.java

```diff
@@ -132,6 +158,8 @@ public static List<ReplicateStatus> getCompletableStatuses() {
                 RESULT_UPLOADING,
                 RESULT_UPLOAD_FAILED,
                 RESULT_UPLOADED,
+                CONTRIBUTE_AND_FINALIZE_ONGOING,
```

Comment:
Are we sure as `REVEALING` is not part of this list ?

Reply:
As far as I understand this part, this list should return all "non-uncompletable statuses", which is not the case right now. Also, it is used only for tests in iexec-core.
Let's rework the tests in iexec-core so that we can remove this method.

Reply:
Honestly, I would go for the consistency in a first step and to go for the cleanup in a second one.

Reply:
Are you sure about your "non-uncompletable" ?
For me it is the list of states allowing to go to `COMPLETING` state which are
* `REVEALED` (worker part of the consensus)
* `CONTRIBUTE_AND_FINALIZE_DONE` (worker has revealed and has uploaded its results)
* `RESULT_UPLOAD` something as the worker is part of the consensus and if this worker fails, to upload, another one could do it (to check if it is really supported)

Reply:
Please have a look at the only use of this method to understand what I mean:
https://github.com/iExecBlockchainComputing/iexec-core/blob/1e6d93d197b0346c4b46a62278d5b7286e56dd0f/src/test/java/com/iexec/core/replicate/ReplicateListenersTests.java#L164-L175

Reply:
After some discussions, we've decided to remove `CONTRIBUTE_AND_FINALIZE_ONGOING` from this list.
Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/408/commits/b585b98c2e2bbfcbad3f0ad11196c0390dcb8849.